### DISCUSSION TITLE
给列头设置了border后，单元格不能拖拽列宽了，并且经常报错

### DISCUSSION THREAD

**Discussion by Create-Peace** [antvis/S2#2602] [Q&A] [answered]
给列头自定义border后，单元格不能拖拽列宽了，并且经常报错
https://stackblitz.com/edit/react-36gzsx?file=src%2Findex.js,package.json
调整demo视图大小会报错
![image](https://github.com/antvis/S2/assets/12973954/4c8d7317-698f-4157-88e6-5af93be42608)

**Comment by lijinke666** [comment, marked as answer]
基类的 `getStyle` 有一个可选参数, 你如果重写的话需要透传, 不然拖拽热区的颜色拿不到, 这里的[文档有误](https://s2.antv.antgroup.com/api/basic-class/base-cell), 后续修改下

https://github.com/antvis/S2/blob/00bb61020cd8a65ca39ce426e9e2b783a90287ac/packages/s2-core/src/cell/base-cell.ts#L318-L322

```diff
class CustomColCell extends ColCell {
-  getStyle() {
-    const defaultStyle = super.getStyle()
-  }

+  getStyle(name) {
+    const defaultStyle = super.getStyle(name)
+  }
}
```

**Reply by Create-Peace** [reply]
非常感谢
